### PR TITLE
ImageCropper v2.5.0

### PR DESCRIPTION
**Pull request by tuyen-vuduc**
Update to ImageCropper v2.5.0

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 NAXAM CO., LTD
```
Comment (mikescandy) [inline LICENSE]: I appreciate using MIT license, but considering this is a pull request you shouldn't attribute to yourself.
Please add mikescandy

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Image Cropper for Xamarin.Android
+A Xamarin.Android binding library for [Image Cropper](https://github.com/ArthurHub/Android-Image-Cropper) library.
+
+```
+Install-Package XamarinAndroidImageCropper
+```
+
+## Usage
+
+Plz follow its origin guide [here](https://github.com/ArthurHub/Android-Image-Cropper/blob/master/README.md).
```
Comment (mikescandy) [inline README.md]: Use proper english
"Please follow the Android library usage guide here"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Contributors
+- [Tuyen Vu](https://vuductuyen.com) @ [NAXAM](https://naxam.net)
```
Comment (mikescandy) [inline README.md]: add - [mikescandy]

File: XamarinAndroidImageCropper/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -6,13 +6,13 @@
...
-[assembly: AssemblyTitle("ClassLibrary1")]
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyTitle("Naxam.ImageCropper.Droid")]
+[assembly: AssemblyDescription("Xamarin.Android binding library - ImageCropper")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("ClassLibrary1")]
-[assembly: AssemblyCopyright("Copyright ©  2016")]
-[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCompany("NAXAM COMPANY LIMITED")]
```
Comment (mikescandy) [inline XamarinAndroidImageCropper/Properties/AssemblyInfo.cs]: As above, why are you attributing this to yourself?

**Review by mikescandy — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (tuyen-vuduc) [reply LICENSE]: Alright, let me update. It's our default template.

File: README.md (hunk 2 above)
Comment (tuyen-vuduc) [reply README.md]: :). Thanks.

File: README.md (hunk 3 above)
Comment (tuyen-vuduc) [reply README.md]: Alright, let me update.

**Comment by tuyen-vuduc** [comment]
I made changes. Plz review again

**Comment by mikescandy** [comment]
I seriously don't understand why you had to put yourself in all the copyright texts, plus setting you as owner and your github as project site. 
Try again to fix it, but I'm considering to close this.

**Comment by mikescandy** [comment]
i'm publishing an updated version, will close this.

**Pull request closed without merging**